addons/pos_event/static/src/app/components/popup/event_configurator_popup/event_configurator_popup.js

```diff
@@ -32,10 +32,10 @@ export class EventConfiguratorPopup extends Component {
     }
     get dialogTitle() {
         const event = this.props.tickets[0].event_id;
-        let title = _t("Select tickets for %s", [event.name]);
+        let title = _t("Select tickets for %s", event.name);
 
         if (event.seats_limited) {
-            title += _t(" (%s seats available)", [event.seats_available]);
```

Comment:
Unrelated to current PR but translation phrase shouldn't be build in parts like this, there should be 2 different versions to translated with or without the part.

Reply:
You're right. Since we're in master, I corrected it 😉